packages/cli/test/integration/sample-project.test.js

```diff
@@ -0,0 +1,159 @@
+const path = require('path');
+const {
+  loadEnvironment,
+  deployOnEnvironment,
+} = require('@synthetixio/deployer/test/helpers/use-environment');
+const assert = require('assert/strict');
+
+// Handy configs for developing on this file:
+const SHOW_CLI_OUTPUT = false; // CI needs this to be false
+const DEPLOY_INSTANCE = true; // CI needs this to be true
+
+describe('sample-project', function () {
```

Comment:
This test is pretty hacky, but IMO it's ok to tolerate some hackyness since it is testing an interactive script.

The idea is to use `child_process` to start the CLI, and interact with it via its stdin and stdout.